packages/core/src/utils.js

```diff
@@ -1,2 +1,19 @@
 // @flow
 export let isBrowser = typeof document !== 'undefined'
+
+const globalKey = '__EMOTION_CORE__'
+const globalContext = isBrowser ? window : global
+
+export function checkForMultipleVersions() {
```

Comment:
this would have to be called to actually make it work :p however, calling this as a function could lead to this not being properly removed in production, so I would just move this function's content to `core/src/index.js` and this should be good to go 👍